main/main.cpp

```diff
@@ -2043,6 +2043,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		BLOCK_DEVICE("AMD", "Radeon (TM) R9 M3");
 
 		// Intel GPUs.
+		BLOCK_DEVICE("Intel", "__ID_CHECK__:Intel(R) HD Graphics"); // Enable device ID checks for drivers using generic "Intel(R) HD Graphics" as name.
+		BLOCK_DEVICE("Intel", "__ID_CHECK__:Intel HD Graphics"); // Enable device ID checks for drivers using generic "Intel HD Graphics" as name.
```

Comment:
Do we even need to do these checks? Are there Intel GPUs which use that generic name and wouldn't benefit from defaulting to ANGLE?

Reply:
I think generic names can happen for some Kaby Lake CPUs, and we currently do not force ANGLE on these. Anything newer should be `UHD Graphics`. But maybe simply adding `Intel HD Graphics` and `Intel(R) HD Graphics` and removing all ID check is OK, since generic names usual happen on low end devices.

Reply:
I have a Kaby Lake and I confirm the name is NOT UHD Graphics but some variant of Intel HD Graphics

Reply:
Kaby Lake CPUs with "numbered" IGPs (the high-end ones) are typically Intel HD Graphics 620 or 630. The "unnumbered" IGPs (the low-end ones) use just "Intel HD Graphics".

Reply:
8th gen should be "Kaby Lake R". My i7-8550U has UHD 620.